src/algebra/group/units.lean

```diff
@@ -22,6 +22,8 @@ structure units (α : Type u) [monoid α] :=
 (val_inv : val * inv = 1)
 (inv_val : inv * val = 1)
 
+postfix `ˣ`:1000 := units
```

Comment:
This should be in the module docstring. Can you give some explanation of where this notation comes from? (i.e. "resembles $some_latex$" or "Mˣ is M with 0 crossed out" or "as used by <some reference>")